src/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj

```diff
@@ -14,6 +14,11 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.3'">
```

Comment:
This isn't necessary as this project doesn't build a netstandard1.3 build configuration.